axum-extra/src/response/multiple.rs

```diff
@@ -0,0 +1,296 @@
+//! Generate forms to use in responses.
+
+use axum::response::{IntoResponse, Response};
+use fastrand;
+use http::{header, HeaderMap, StatusCode};
+use mime::Mime;
+
+/// Create multipart forms to be used in API responses.
+///
+/// This struct implements [`IntoResponse`], and so it can be returned from a handler.
+#[derive(Debug)]
+pub struct MultipartForm {
+    parts: Vec<Part>,
+}
+
+impl MultipartForm {
+    /// Initialize a new multipart form with the provided vector of parts.
+    ///
+    /// # Examples
+    ///
+    /// ```rust
+    /// use axum_extra::response::multiple::{MultipartForm, Part};
+    ///
+    /// let parts: Vec<Part> = vec![Part::text("foo".to_string(), "abc"), Part::text("bar".to_string(), "def")];
+    /// let form = MultipartForm::with_parts(parts);
+    /// ```
+    #[deprecated]
```

Comment:
I just found this for another reason... Why did we add a new deprecated item here?? @zleyyij

Reply:
That definitely shouldn't be there, I'm so sorry. I probably added it during a refactor with the intention of changing that method before deciding it would be better if I didn't.

I'll remove it.

Thank you so much for your patience, I'm so sorry about all the hassle this PR has created

Reply:
Will be resolved by https://github.com/tokio-rs/axum/pull/2957